src/main/java/com/rtb/projectmanagementtool/auth/AuthOps.java

```diff
@@ -27,7 +27,8 @@ public AuthOps(DatastoreService ds) {
     UserController controller = new UserController(ds);
   }
 
-  public void loginUser(HttpServletRequest request, HttpServletResponse response) {
+  // generates cookie. Used in most methods.
+  public Cookie generateCurrCookie(HttpServletRequest request, HttpServletResponse response) {
```

Comment:
Good reusable method.  Although it doesn't really generate a cookie, but more so it extracts the cookie from the request.  Maybe something like "getCookieFromRequest" would work better.

(Also if the response is not needed, please remove from the params)